### ISSUE TITLE
IndexSizeError after inserting image and then selecting everything in the editor

### ISSUE THREAD

**Issue by anna-kartynnik** [reedsy/quill-cursors#57]
Hello, thank you for your module. I found a strange issue while working with it.

Steps to reproduce:

1. Open https://jsfiddle.net/ganachka/w0udaost/5/ (I used the same code you have in your example folder).
2. Additionally open Developer tools with browser console.
3. Paste an image to one of the editors (it's important to copy image itself and paste it, not its URL).
4. Select the content of the editor, for example by pressing Cmd+A

After that I get `IndexSizeError: Index or size is negative or greater than the allowed amount` from the `quill-cursors` script. I also get the same error when I resize browser window after that.

The screenshot from Firefox (but experienced the same in Chrome too):

<img width="1676" alt="Screen Shot 2020-05-20 at 12 11 34 AM" src="https://user-images.githubusercontent.com/15210801/82416200-80e6bd80-9a2e-11ea-8235-50d139c2241d.png">

Do you have any ideas about how it can be fixed?

**Comment by alecgibson** [comment]
Interesting, if I follow your steps I get an error, but a different error:

```
quill-cursors.js:1 Uncaught DOMException: Failed to execute 'setEnd' on 'Range': There is no child at offset 1.
    at https://unpkg.com/quill-cursors@3.0.0/dist/quill-cursors.js:1:11573
    at Array.reduce (<anonymous>)
    at t._lineRanges (https://unpkg.com/quill-cursors@3.0.0/dist/quill-cursors.js:1:11299)
    at t._updateCursor (https://unpkg.com/quill-cursors@3.0.0/dist/quill-cursors.js:1:10035)
    at t.moveCursor (https://unpkg.com/quill-cursors@3.0.0/dist/quill-cursors.js:1:8510)
    at https://fiddle.jshell.net/ganachka/w0udaost/5/show/?editor_console=:190:30
```

I'll investigate

**Comment by alecgibson** [comment]
@anna-kartynnik released a fix in `3.0.1`. Please let me know if this doesn't solve your issue.

**Comment by anna-kartynnik** [comment]
Works for me, thanks!
